src/test/java/tech/jhipster/lite/generator/ci/renovate/RenovateModuleFactoryTest.java

```diff
@@ -0,0 +1,47 @@
+package tech.jhipster.lite.generator.ci.renovate;
+
+import static tech.jhipster.lite.module.infrastructure.secondary.JHipsterModulesAssertions.assertThatModuleWithFiles;
+import static tech.jhipster.lite.module.infrastructure.secondary.JHipsterModulesAssertions.pomFile;
+import static tech.jhipster.lite.module.infrastructure.secondary.JHipsterModulesAssertions.readmeFile;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.junit.jupiter.MockitoExtension;
+import tech.jhipster.lite.TestFileUtils;
+import tech.jhipster.lite.UnitTest;
+import tech.jhipster.lite.generator.ci.renovate.domain.RenovateModuleFactory;
+import tech.jhipster.lite.module.domain.JHipsterModule;
+import tech.jhipster.lite.module.domain.JHipsterModulesFixture;
+import tech.jhipster.lite.module.domain.properties.JHipsterModuleProperties;
+
+@UnitTest
+@ExtendWith(MockitoExtension.class)
+class RenovateModuleFactoryTest {
+
+  @InjectMocks
+  private RenovateModuleFactory factory;
```

Comment:
There's no dependency here, so no need for Mockito, we can just create a new instance using the constructor: 
```suggestion
  private RenovateModuleFactory factory = new RenovateModuleFactory();
```

Reply:
Nice catch. Thanks.